### PR TITLE
[Console-IoT] Fix count card in project list

### PR DESCRIPTION
**Pull request by suyash-naithani**
### Type of change

- Bugfix

### Description

 Fix count card in project list

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: console/console-init/ui/src/modules/project/ProjectPage.tsx [hunk 1]
```diff
@@ -69,9 +69,7 @@ export default function ProjectPage() {
   const [msgCount, setMsgCount] = useState<IProjectCount>(
     setInitialProjcetCount()
   );
-  const [ioTCount, setIoTCount] = useState<IProjectCount>(
-    setInitialProjcetCount()
-  );
+  const [ioTCount, setIoTCount] = useState(setInitialProjcetCount());
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/project/ProjectPage.tsx:72]: Typo 'setInitialProjectCount'

File: console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx [hunk 2]
```diff
@@ -83,6 +88,15 @@ export const ProjectListContainer: React.FC<IProjectListContainerProps> = ({
     setDeleteIoTProjectQueryVariables
   ] = useMutationQuery(DELETE_IOT_PROJECT, ["allProjects"]);
 
+  var project_count = useQuery<IAllProjectsResponse>(RETURN_COUNT_PROJECTS(), {
+    fetchPolicy: FetchPolicy.NETWORK_ONLY,
+    pollInterval: POLL_INTERVAL
+  });
+
+  const totalProjects = (project_count && project_count.data) || {
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx:96]: Can you please rename totalProjects to allProjects or something similar, as total project gives me an indication that it's the count of the projects.

File: console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx [hunk 3]
```diff
@@ -252,8 +266,10 @@ export const ProjectListContainer: React.FC<IProjectListContainerProps> = ({
     });
   };
 
-  const getProjects = () => {
-    return projects?.map((project: any) => {
+  const getProjects = (
+    projects: Array<IAddressSpaceType | IIotProjectType>
+  ) => {
+    return projects.map((project: any) => {
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx:272]: Please avoid using 'any' and assign a type for project.

File: console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx [hunk 4]
```diff
@@ -287,52 +303,55 @@ export const ProjectListContainer: React.FC<IProjectListContainerProps> = ({
     });
   };
 
-  const projectList: IProject[] = getProjects();
+  const projectList: IProject[] = getProjects(projects);
+
+  const project_info =
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx:308]: Please use camel casing.

**Review by anukritijha — CHANGES_REQUESTED** [review]

File: console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx (hunk 2 above)
Comment (suyash-naithani) [reply console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx:96]: allProjects is already defined, how about projectsList?

**Review by ajaypratap003 — APPROVED** [review]

**Review by drcoolsanjeev — APPROVED** [review]

**Review by anukritijha — APPROVED** [review]

**Pull request merged**